Branch-SDK/src/io/branch/referral/ServerRequestRegisterView.java

```diff
@@ -106,6 +107,10 @@ private JSONObject createContentViewJson(BranchUniversalObject universalObject,
 
         paramsObj.put(Defines.Jsonkey.ContentKeyWords.getKey(), universalObject.getKeywordsJsonArray());
         paramsObj.put(Defines.Jsonkey.PublicallyIndexable.getKey(), universalObject.isPublicallyIndexable());
+        if (!universalObject.getCurrencyType().equals(CURRENCY_TYPE.NONE.toString())) {
+            paramsObj.put(Defines.Jsonkey.PurchaseAmount.getKey(), universalObject.getPrice());
```

Comment:
I think we should do these separately, and note that we default to USD in currency.